### DISCUSSION TITLE
Add option to disable confirmation of removal tabs

### DISCUSSION THREAD

**Discussion by tikodeur** [Chatterino/chatterino2#4716] [Ideas]
Hello,

Currently, closing a tab (whether it's the active tab or another) always comes with a confirmation of removal, unless you are actively on the tab and therefore you can press `Ctrl`+`Shift`+`W`.

It would be great to have the possibility to disable this confirmation in the settings (with the confirmation being activated by default), allowing closing of a tab with a simple click, whether it's using the cross or the middle/scroll button.
<details>
  <summary></summary>
  
_PS: didn't know if I should post in this section or in the enhancement issues section._
  
</details>

**Comment by Felanbird** [comment]
This is an interesting one because you are more or less asking for the opposite of what people requested for years.

People always wanted a confirmation for pressing said keybinds, discussed in many issues but going all the way back to #339

Due to the fact that this was the majority problem, the ability to re-open the tab with `ctrl+G` ended up being the resolution, instead of a blanket enable/disable for the buttons & keybinds.

**Reply by tikodeur** [reply]
I don't think it's the opposite of the current feature (which would be the removal of confirmation). I'd like to have the choice to activate or deactivate the confirmation if I want.

I just want to directly close the tab when I want it, especially when I'm pressing the middle button of my mouse, which in this case is anything but a misclick. 

My issue is that, if I want to directly close a tab without the pop-up window, I have to either activate the tab then press `Ctrl`+`Shift`+`W` given that the hotkey doesn't have the confirmation, or use right-click + "Close the tab".

> the ability to re-open the tab with ctrl+G ended up being the resolution

Btw, the `Ctrl`+`G` hotkey reopens the split, not the tab. I am unable to reopen tabs with this hotkey, which is the intended functioning.

**Comment by x9136** [comment]


**Reply by pajlada** [reply]
You can already disable the close button on tabs (this is a setting)

**Reply by x9136** [reply]
@pajlada Great, but if I need to close tabs regularly, I'll enable the option. The misclick problem will return. I'll want to go to another tab and by doing so I'll accidentally try to close it. Fitts's law.

**Reply by pajlada** [reply]
If you need to close tabs regularly, you can right-click a tab and press remove. You can also middle-mouse-click the tab to remove it. You should not need to toggle the setting on and off
